code/modules/mob/living/carbon/monkey/punpun.dm

```diff
@@ -1,12 +1,13 @@
 /mob/living/carbon/monkey/punpun //except for a few special persistence features, pun pun is just a normal monkey
 	name = "Pun Pun" //C A N O N
 	unique_name = 0
+	/// If we had one of the rare names in a past life
 	var/ancestor_name
+	/// This does not represent how many rounds Pun Pun has survived, instead it increases each time he dies, and resets when he gibs
```

Comment:
```suggestion
	/// The number of times Pun Pun has died. Resets when he is gibbed.
```

We don't really need to be documenting what things *aren't*.

Reply:
done